src/Synapse/Synapse/help/Set-AzSynapseSqlAuditSetting.md

```diff
@@ -43,82 +47,94 @@ Set-AzSynapseSqlAuditSetting [-AuditActionGroup <AuditActionGroups[]>] [-Predica
 ```
 
 ## DESCRIPTION
+
 The **Set-AzSynapseSqlAuditSetting** cmdlet changes the auditing settings of an Azure Synapse Analytics Workspace.
 When blob storage is a destination for audit logs, specify the *StorageAccountResourceId* parameter to determine the storage account for the audit logs and the *StorageKeyType* parameter to define the storage keys. You can also define retention for the audit logs by setting the value of the *RetentionInDays* parameter to define the period for the audit logs.
 
 ## EXAMPLES
 
 ### Example 1
+
 ```powershell
-Set-AzSynapseSqlAuditSetting -WorkspaceName ContosoWorkspace -BlobStorageTargetState Enabled -StorageAccountResourceId "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/resourcegroup01/providers/Microsoft.Storage/storageAccounts/mystorage" -StorageKeyType Primary
+Set-AzSynapseSqlAuditSetting -WorkspaceName ContosoWorkspace -ResourceGroupName ContosoResourceGroup -BlobStorageTargetState Enabled -StorageAccountResourceId "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/resourcegroup01/providers/Microsoft.Storage/storageAccounts/mystorage" -StorageKeyType Primary
```

Comment:
From the syntax of `Set-AzSynapseSqlAuditSetting`, `ResourceGroupName` is an optional parameter, @wonner , could you help verify current change?

Reply:
Hi @BethanyZhou @isshukla It's a bug need to fix. For now `ResourceGroupName` is a required parameter. @yanjungao718 please help fix, let's keep `ResourceGroupName` as optional.

Reply:
ok, I'll make the change.